src/main/java/dev/emi/trinkets/api/client/TrinketRenderer.java

```diff
@@ -0,0 +1,111 @@
+package dev.emi.trinkets.api.client;
+
+import dev.emi.trinkets.api.Trinket;
+import net.minecraft.client.network.AbstractClientPlayerEntity;
+import net.minecraft.client.render.VertexConsumerProvider;
+import net.minecraft.client.render.entity.model.PlayerEntityModel;
+import net.minecraft.client.util.math.MatrixStack;
+import net.minecraft.client.util.math.Vector3f;
+import net.minecraft.item.ItemStack;
+
+public interface TrinketRenderer {
```

Comment:
We might want to review these translations at some point, perhaps a translate to main arm and secondary arm rather than right and left? Or perhaps both?

Reply:
Hmm, fair point. Since I'd imagine this might be lower priority, should I just mark it with a to-do for now?

Reply:
Sounds good to me, we can discuss in dm at some later point before release.